### PR TITLE
Generate datasets for import

### PR DESCRIPTION
**Pull request by kyleconroy**
Remove the `dbload` command and replace it with `gobike-dataset`, which generates a single CSV suitable for import into Postgres or BigQuery. I've also moved all the geo-related functionality into a separate package.

### CODE REVIEW

**Comment by kyleconroy** [comment]
I've confirmed that the generated CSV can be imported into BigQuery. Also, it takes about 30 seconds to import the data into Postgres via `COPY FROM`, which is way faster than `dbload` ever was.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Datasets
+
+In addition to server, we've made the Ford GoBike datasets available online.
+
+## BigQuery
+
+All trip data lives in the
+[ford_gobike](https://bigquery.cloud.google.com/table/kjc-datasets:ford_gobike.trips)
```
Comment (kyleconroy) [inline README.md]: Happy to put this under a generic project ID if you can think of one.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+dataset, which is availably publicly.
```
Comment (kevinburke) [inline README.md]: available*

File: cmd/gobike-dataset/main.go [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+			"user_type",
+			"member_birth_year",
+			"member_gender",
+			"bike_share_for_all",
+		})
+	}
+
+	for i, trip := range trips {
+		if trip.EndTime.Before(trip.StartTime) {
+			// WTF?
```
Comment (kevinburke) [inline cmd/gobike-dataset/main.go]: Should we panic here so we can identify the bad rows

File: geo/us.go [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+	s2.Point{r3.Vector{-123.32823, 37.673575, 0}},
+	s2.Point{r3.Vector{-123.21929, 38.135944, 0}},
+	s2.Point{r3.Vector{-124.722484, 40.396588, 0}},
+	s2.Point{r3.Vector{-124.360964, 41.486153, 0}},
+	s2.Point{r3.Vector{-124.90688, 42.782148, 0}},
+	s2.Point{r3.Vector{-124.233776, 46.069884, 0}},
+	s2.Point{r3.Vector{-125.003617, 48.495179, 0}},
+}
+
+var alaskaPoints = []s2.Point{
```
Comment (kevinburke) [inline geo/us.go]: Crucial

File: README.md (hunk 1 above)
Comment (kyleconroy) [reply README.md]: I think I'm going to go with `bay-area-mobility`

File: cmd/gobike-dataset/main.go (hunk 3 above)
Comment (kyleconroy) [reply cmd/gobike-dataset/main.go]: We probably want to clean it up when we're parsing the data itself.

**Pull request merged**